cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaLocalizedProduct.js

```diff
@@ -373,6 +373,17 @@ function algoliaLocalizedProduct(parameters) {
                 this[attributeName] = aggregatedValueHandlers[attributeName](product, parameters);
             } else {
                 var config = algoliaProductConfig.attributeConfig_v2[attributeName];
+
+                // if there is no config for the attribute, but it's a custom or activeData attribute, we assume some defaults
+                if (!config && (attributeName.indexOf('custom.') > -1 || attributeName.indexOf('activeData.') > -1)) {
```

Comment:
The goal was to remove the constraint to declare any attributes, not only the activeData or custom ones. Do you foresee any problem to do that?

Reply:
I also think about that and only adding this condition `attributeName.indexOf('.') > -1`, but for more safety, I keep it like that.